README.md

```diff
@@ -0,0 +1,194 @@
+# OpenIddict, as an OpenID Connect Addict
+
+### What's Open Iddict ?
+
+OpenIddict library aims to provide a simple and easy out of the box solution to implement your OpenID Connect 
+server for ASP.NET 5.
+OpenIddict leverages the use of Identity (for user management) and Entityframework (as store provider).
+
+### What's Open ID Connect ?
+
+Open ID is a protocol that allows multiple clients to delegate the Authentication to one or multiple Authority servers.
```

Comment:
nit: `Open ID` -> `OpenID Connect` (OpenID and OpenID Connect are two different specs)